includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -316,7 +316,10 @@ public static function subscription_pricing_fields() {
 				?>
 			</label>
 			<span class="wrap">
-				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php echo esc_attr_x( 'e.g. 5.90', 'example price', 'woocommerce-subscriptions' ); ?>" step="any" min="0" value="<?php echo esc_attr( wc_format_localized_price( $chosen_price ) ); ?>" />
+				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php
+				  // translators: %s the formatted example price value.
+				  echo sprintf( esc_attr_x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), str_replace( get_woocommerce_currency_symbol(), '', strip_tags( wc_price( 5.90 ) ) ) );
```

Comment:
Rather than doing a string replace on the currency symbol can we just use the same function as the one below?

```suggestion
				  echo sprintf( esc_attr_x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), wc_format_localized_price( '5.90' ) );
```

Reply:
Thanks, I searched for a function that to do this but couldn't find, fixed on https://github.com/Automattic/woocommerce-subscriptions-core/pull/807/commits/bd8a70a9a302329f3e0d4fb930bd1c784052ee70.